### ISSUE TITLE
Publish/socialize a roadmap

### ISSUE THREAD

**Issue by laserson** [bigdatagenomics/adam#591]

**Comment by massie** [comment]
Created a `0.17.0` milestone for us to use for this. We can socialize and tracker progress using the milestone features of github.

https://github.com/bigdatagenomics/adam/milestones

**Comment by ryan-williams** [comment]
How set in stone are the issues in [the `0.17.0` milestone](https://github.com/bigdatagenomics/adam/milestones/0.17.0)?

#599 at least seems like it'd take a while to fix in ADAM, since it will need a Spark release after the necessary changes make it in to Spark.

Any interest in doing an `0.16.1` release, or punting some of those issues beyond `0.17.0` so that we can get it out the door sooner? `0.16.0` was > 3mos ago…

**Comment by fnothaft** [comment]
@ryan-williams we're hoping to cut a 0.17.0 "soon" --> hopefully this week. I've just added [0.18.0](https://github.com/bigdatagenomics/adam/milestones/0.18.0) and [1.0.0](https://github.com/bigdatagenomics/adam/milestones/1.0.0) milestones and retagged issues as necessary.

**Comment by heuermh** [comment]
@massie It was discussed in a recent phone call that we should post upcoming release intentions to the developer mailing list.

I created [0.18.2](https://github.com/bigdatagenomics/adam/milestones/0.18.2) and [0.19.0](https://github.com/bigdatagenomics/adam/milestones/0.19.0) milestones, and understand that 0.18.2 will be for fixing issue #872 and that 0.19.0 will be for performance improvements, but I don't have any further details.  Might you be able to write the release intentions post?

I will send the 0.18.1 release announcement once bigdatagenomics/bigdatagenomics.github.io#9 has been merged and the blog updated (or with a little hand holding, I should be able to do that myself).
